### DISCUSSION TITLE
Visual Locators

### DISCUSSION THREAD

**Discussion by sushil-relicx** [getgauge/taiko#1836] [Ideas]
Hi - does Taiko support locate elements based on visually using a image/snapshot, similar to what is being described here: https://applitools.com/docs/features/visual-locators.html

**Comment by zabil** [comment]
Taiko does not have this feature.

**Comment by sriv** [comment]
You can try using https://github.com/saikrishna321/taiko-eyes, which integrates taiko with applitool eyes.
